overlord/snapstate/autorefresh.go

```diff
@@ -157,8 +157,14 @@ func canRefreshOnMeteredConnection(st *state.State) (bool, error) {
 	if err != nil && err != state.ErrNoState {
 		return false, err
 	}
-
-	return onMetered != "hold", nil
+	switch onMetered {
+	case "hold":
+		return false, nil
+	case "force":
+		return true, nil
+	default:
```

Comment:
Should the default case be an error? After all there are only three valid values, "", "force" and "hold"